spec/models/job_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+
+describe Job do
+  it { should belong_to(:organization) }
+
+  before(:each) do
+    @organization = create(:organization)
+    @job1 = create(:job, organization_id: @organization.id, expires_at: DateTime.now + 1.day)
+    @job2 = create(:job, organization_id: @organization.id, expires_at: DateTime.now - 1.day)
+    @job3 = create(:job, organization_id: @organization.id)
+  end
+
+  describe 'active scope' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.